content/blog/2019/06/2019-06-25-phase-1-multibranch-pipeline-support-for-gitlab.adoc

```diff
@@ -0,0 +1,324 @@
+---
+layout: post
+title: "Multi-branch Pipeline Jobs Support for GitLab SCM"
+tags:
+- gitlab
+- plugins
+- pipeline
+- credentials
+- developer
+- gsoc
+- gsoc2019
+author: baymac
+---
+
+= Multi-branch Pipeline Support for GitLab SCM
+
+This is one of the Jenkins project in GSoC 2019. We are working on adding support for link:/projects/gsoc/2019/gitlab-support-for-multibranch-pipeline/[Multi-branch Pipeline Jobs and Folder Organisation in GitLab]. The plan is to create the following
+plugins:
+
+1. link:https://github.com/jenkinsci/gitlab-api-plugin[GitLab API Plugin] - Wraps GitLab Java API
+
+2. link:https://github.com/baymac/gitlab-branch-source-plugin[GitLab Branch Source Plugin]:
+
+    Contains two packages:
+ 
+     * `io.jenkins.plugins.gitlabserverconfig` - Manages server configuration and web hooks management. Ideally should reside inside
+     another plugin with name `GitLab Plugin`. In future, this package should be moved into a new plugin.
+     
+     * `io.jenkins.plugins.gitlabbranchsource` - Adds GitLab Branch Source for Multi-branch Pipeline Jobs (including
+     Merge Requests) and Folder organisation.
+
+== Jenkins and GitLab Integration:
+
+This section contains information about Jenkins CI support for GitLab.
+
+=== Present State
+
+1. FreeStyle Job and Pipeline(Single Branch) Job are fully supported.
```

Comment:
```suggestion
* FreeStyle Job and Pipeline(Single Branch) Job are fully supported.
```

General guidance: Use bulleted lists unless you are listing steps or things that must happen in a specific order.